### PR TITLE
Ng organizer section

### PR DESCRIPTION
**Pull request by Nikolay-Gavazov**
Little CSS changes.

### CODE REVIEW

**Review by Getsov — DISMISSED** [review]

File: src/app/components/user/forgotten-password/forgotten-password.component.scss [hunk 1]
```diff
@@ -49,3 +49,19 @@
     cursor: pointer;
   }
 }
+
+
+@media (max-width: 430px) {
+    .container{
```
Comment (Getsov) [inline src/app/components/user/forgotten-password/forgotten-password.component.scss:55]: put spaces everywhere before the opening bracket "{"

File: src/app/components/user/forgotten-password/forgotten-password.component.scss [hunk 2]
```diff
@@ -49,3 +49,19 @@
...
+        width: 91%;
+        .buttons{
+            margin-top: 24px;
+            .secondary{
+                width: 100%;
+            }
+            .submit{
+                width: 100%;
```
Comment (Getsov) [inline src/app/components/user/forgotten-password/forgotten-password.component.scss:63]: Isn't this the same style as .secondary ?

File: src/shared/components/section-sort/section-sort.component.scss [hunk 3]
```diff
@@ -202,3 +202,20 @@
     }
   }
 }
+
+@media (max-width: 430px) {
+    .section{
```
Comment (Getsov) [inline src/shared/components/section-sort/section-sort.component.scss:207]: Same here for the opening brackets.

File: src/shared/components/section-sort/section-sort.component.scss [hunk 4]
```diff
@@ -202,3 +202,20 @@
...
+        width: 91%;
+        .select-section{
+            width: 100%;
+            flex-direction: column;
+            gap:0;
```
Comment (Getsov) [inline src/shared/components/section-sort/section-sort.component.scss:212]: Add space before the zero (0)

File: src/shared/components/section-sort/section-sort.component.scss [hunk 5]
```diff
@@ -202,3 +202,20 @@
...
+            .select-item{
+                width: 100%;
+            }
+        }
+    }
+    ::ng-deep .alert-wrapper{
```
Comment (Getsov) [inline src/shared/components/section-sort/section-sort.component.scss:218]: Is this the only way working? No other ways?

File: src/shared/components/section-sort/section-sort.component.scss (hunk 4 above)
Comment (Nikolay-Gavazov) [reply src/shared/components/section-sort/section-sort.component.scss:212]: done

File: src/shared/components/section-sort/section-sort.component.scss (hunk 3 above)
Comment (Nikolay-Gavazov) [reply src/shared/components/section-sort/section-sort.component.scss:207]: done

File: src/app/components/user/forgotten-password/forgotten-password.component.scss (hunk 1 above)
Comment (Nikolay-Gavazov) [reply src/app/components/user/forgotten-password/forgotten-password.component.scss:55]: done

File: src/shared/components/section-sort/section-sort.component.scss (hunk 5 above)
Comment (Nikolay-Gavazov) [reply src/shared/components/section-sort/section-sort.component.scss:218]: without ::ng-deep doesnt work

**Review by Getsov — APPROVED** [review]

**Pull request merged**